package.json

```diff
@@ -24,6 +24,7 @@
     "build": "node auspice.js build --verbose",
     "prepare": "npm run build",
     "lint": "eslint src",
+    "fix": "eslint --fix src",
```

Comment:
I like the pattern of using `npm run fix` to cleanup linting errors, can understand if you'd like to keep this out of the PR.

Reply:
Happy for this to be here but please rename to `lint:fix`